### ISSUE TITLE
Keyboard navigation is broken when results grid is scrolled down

### ISSUE THREAD

**Issue by insmac** [questdb/ui#227]
Steps to replicate:

1. Generate a result set big enough to allow for some scrolling.
2. Try using keyboard arrows (top/bottom, left/right) to see that the correct field is highlighted.
3. Scroll down, and try doing the same - keyboard navigation seems to be broken: sometimes it goes only horizontally, while up/down doesn't work, and sometimes it loses focus on the arrow up/down.

Screen recording:

https://github.com/questdb/ui/assets/1871646/02806f82-fec7-4a56-8032-485963fed9e1
